content/en/synthetics/cicd_integrations/gitlab.md

```diff
@@ -0,0 +1,101 @@
+---
+title: GitLab
+kind: documentation
+description: Configure your GitLab instance to run Synthetic tests in your CI/CD pipelines.
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+  tag: "Blog"
+  text: "Run Datadog Synthetic tests in your GitLab pipelines"
+- link: "/continuous_integration/setup_pipelines/gitlab/"
+  tag: "Documentation"
+  text: "Set up Tracing on a GitLab Pipeline"
+---
+
+## Overview
+
+Run Synthetic tests in your [GitLab][1] pipelines, block deployments, and trigger rollbacks to ensure your code is added in production when your essential business workflows are functioning as expected.
```

Comment:
The hyperlink for GitLab links to the integration page which uses a lower case L (`Gitlab`), should we ask the team to update it?

Reply:
I can open a PR to update the case 👍